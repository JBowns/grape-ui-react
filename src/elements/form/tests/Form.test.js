import React from 'react';
import { Form } from '../index';
import 'jest-styled-components';
// import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { configure, mount } from 'enzyme';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

describe('Form Component base', () => {
  it('should return a Form object without class inline', () => {
    const component = mount(<Form />);
    expect(toJson(component.find('Form'))).toMatchSnapshot();
  });
  it('should return a Form object with class inline', () => {
    const component = mount(<Form formInline />);
    expect(toJson(component.find('Form'))).toMatchSnapshot();
  });
});
