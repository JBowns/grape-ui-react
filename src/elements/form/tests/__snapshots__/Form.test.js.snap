// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Component base should return a Form object with class inline 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: auto;
}

<Form
  formInline={true}
>
  <styled.form
    className="form-inline"
  >
    <form
      className="form-inline c0"
    />
  </styled.form>
</Form>
`;

exports[`Form Component base should return a Form object without class inline 1`] = `
<Form
  formInline={false}
>
  <styled.form
    className=""
  >
    <form
      className=""
    />
  </styled.form>
</Form>
`;
